extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -210,4 +223,35 @@ public int determineSslPort(LaunchMode launchMode) {
         REDIRECT,
         DISABLED;
     }
+
+    public enum ForwardedHostHeader {
+        X_FORWARDED_HOST("X-Forwarded-Host"),
+        X_FORWARDED_SERVER("X-Forwarded-Server"),
+        REFERER("Referer");
+
+        AsciiString header;
+
+        ForwardedHostHeader(String name) {
+            header = AsciiString.cached(name);
+        }
+
+        public AsciiString getHeaderName() {
```

Comment:
Please have both enums have the same method name, `getHeaderName` or `getHeader`

Reply:
Done